xarray/core/dataarray.py

```diff
@@ -4418,10 +4418,34 @@ def query(
         )
         return ds[self.name]
 
+    def drop_duplicates(
+        self,
+        subset: Optional[Union[Hashable, Sequence[Hashable]]] = None,
```

Comment:
Could this be called `dims` / `coords`?
No need for the `Optional` in the default is `None` FYI

Reply:
Thoughts on following pandas lead vs dims/coords?
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop_duplicates.html

Reply:
I would vote to be consistent within xarray, rather than across libraries for this method.